GameServer.py

```diff
@@ -6,35 +6,44 @@
 from PodSixNet.Server import Server
 
 from ClientHandler import ClientHandler
-
+from PlayerInfo import PlayerInfo
 
 class GameServer(Server):
     channelClass = ClientHandler
+    idArray = [0, 0, 0]
+    playersDict = []
 
     def __init__(self, *args, **kwargs):
         Server.__init__(self, *args, **kwargs)
         self.players = WeakKeyDictionary()
         print('Server launched')
 
     def AddPlayer(self, player):
-        self.players[player] = True
-        print("New Player: " + str(player.addr))
-        print("Connected players: ", [p for p in self.players])
+        playerinfo = PlayerInfo(0,0, self.getNewId())
+        self.players[player] = playerinfo
+        self.playersDict.append({"id": playerinfo.id, "x": playerinfo.x, "y": playerinfo.y})
+
+    def getNewId(self):
+        for i in range(0, 2):
+            if self.idArray[i] == 0:
+                self.idArray[i] = 1
+                return i+1
 
     def DelPlayer(self, player):
         print("Deleting Player" + str(player.addr))
         del self.players[player]
 
     def SendToAll(self, data):
-        [p.Message(data) for p in self.players]
+        [p.PlayersDict(data) for p in self.players]
 
```

Comment:
What does this function send? If we're sending players data then maybe `SendAllPlayersDataToAll()` would be a better choice

Reply:
I still dont know why we're naming methods with uppercase, we should change this two and only stick with `Network_function()` naming with this uppercase at the start

Reply:
Yep, I changed name to SendAllPlayersDataToAll()

Yes, I changed methods names except "Connected", which needs to be with uppercase as well as Network as you mentioned